pulp_rpm/app/models/advisory.py

```diff
@@ -205,7 +205,9 @@ def get_pkglist(self):
         """
         pkglist = []
         for collection in self.collections.all():
-            for pkg in collection.packages.all():
+            for pkg in collection.packages.all().order_by(
+                "name", "epoch", "version", "release", "arch"
```

Comment:
Could we order by checksum or perhaps location href?

Reply:
Yes, either of those would work I think, will investigate

Reply:
+1.  Would much rather sort by as few fields as possible.